test/test_onnx.py

```diff
@@ -76,11 +87,34 @@ def test_non_float_params(self):
         torch._C._jit_pass_onnx(trace)
         self.assertONNXExpected(trace.export(initializers))
 
+    def test_symbolic_mismatch(self):
+        class MyFun(Function):
+            @staticmethod
+            def symbolic(g, x):
+                assert False  # dead code
+
+            @staticmethod
+            def forward(ctx, x, y):
+                return x + y
+
+        class MyModel(Module):
+            def forward(self, x, y):
```

Comment:
We're doing so many of these `MyModel`s that we should probably have a helper that takes a lambda and wraps it in a module